vpr/src/tileable_rr_graph/tileable_rr_graph_edge_builder.cpp

```diff
@@ -119,46 +120,173 @@ void build_rr_graph_edges(const RRGraphView& rr_graph,
                           const bool& perimeter_cb,
                           const bool& opin2all_sides,
                           const bool& concat_wire,
-                          const bool& wire_opposite_side) {
+                          const bool& wire_opposite_side,
+                          const RRSwitchId& delayless_switch,
+                          const bool& is_vib_arch) {
+    /* Create map from medium mux name to index */
+    std::vector<std::vector<std::vector<std::map<std::string, size_t>>>> medium_mux_name2medium_index;
+
+    if (is_vib_arch) {
+        medium_mux_name2medium_index.resize(vib_grid.dim_size(0));
```

Comment:
Replace the ``dim_size(0)`` and ``dim_size(1)`` with APIs ``height()`` and ``width()``